### PR TITLE
Add secrets to SQL examples

### LINKED ISSUE #823: Show how to use secrets with SQL queries
We need to add examples showing how to use secrets to securely populate sensitive credentials in queries that pull data from authenticated data stores (MySQL, Postgres, etc.)

##### Relevant URLs
- [x] https://v2.docs.influxdata.com/v2.0/query-data/flux/sql/
- [x] https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/sql/from/
- [x] https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/sql/to/
- [x] https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/experimental/bigtable/from/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #823 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/query-data/flux/sql.md [hunk 1]
```diff
@@ -154,6 +155,77 @@ Use the variable to manipulate queries in your dashboards.
 
 ---
 
+## Use secrets to populate connection credentials
+If your SQL database requires authentication, use [InfluxDB secrets](/v2.0/security/secrets/)
+to store and populate sensitive connection credentials for your database.
```
Comment (pierwill) [inline content/v2.0/query-data/flux/sql.md:160]: Maybe: "to store and populate ~sensitive~ connection credentials ~for your database~."

File: content/v2.0/query-data/flux/sql.md [hunk 2]
```diff
@@ -154,6 +155,77 @@ Use the variable to manipulate queries in your dashboards.
...
+By default, InfluxDB encrypts and stores secrets in its internal key-value store, BoltDB,
+but also integrates directly with [Vault](https://www.vaultproject.io/).
```
Comment (pierwill) [inline content/v2.0/query-data/flux/sql.md:162]:
```
By default, InfluxDB stores secrets in its internal key-value store, BoltDB.
For added security, use [Vault](https://www.vaultproject.io/).
```

Comment (pierwill) [reply content/v2.0/query-data/flux/sql.md:162]: (BoltDB does not encrypt data, so let's encourage people to use Vault for projects that involve authentication.)

File: content/v2.0/query-data/flux/sql.md [hunk 3]
```diff
@@ -154,6 +155,77 @@ Use the variable to manipulate queries in your dashboards.
...
+
+_For information about using **Vault** with InfluxDB, see [Setup a Vault cluster](/v2.0/security/secrets/use-vault/)._
+
+### Store your database credentials as secrets
+Use the **InfluxDB API** or the **`influx` CLI** to store your database credentials as secrets.
```
Comment (pierwill) [inline content/v2.0/query-data/flux/sql.md:167]: Links for these bolded items?

File: content/v2.0/query-data/flux/sql.md [hunk 4]
```diff
@@ -154,6 +155,78 @@ Use the variable to manipulate queries in your dashboards.
 
 ---
 
+## Use secrets to populate connection credentials
+If your SQL database requires authentication, use [InfluxDB secrets](/v2.0/security/secrets/)
+to store and populate connection credentials.
+By default, InfluxDB base64-encodes and stores secrets in its internal key-value store, BoltDB.
+For added security, [Vault](https://www.vaultproject.io/).
+
+_For information about the **InfluxDB Vault integration**, see [Store secrets in Vault](/v2.0/security/secrets/use-vault/)._
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/sql.md:164]: For added security, [store secrets in Vault](/v2.0/security/secrets/use-vault/).

File: content/v2.0/query-data/flux/sql.md [hunk 5]
```diff
@@ -154,6 +155,78 @@ Use the variable to manipulate queries in your dashboards.
...
+  -d '{
+  "POSTGRES_HOST": "http://example.com",
+  "POSTGRES_USER": "example-username",
+  "POSTGRES_PASS": "example-password"
+}'
+```
+
+_**Helpful links**_  
+[View your organization ID](/v2.0/organizations/view-orgs/#view-your-organization-id)  
+[View your authentication token](/v2.0/security/tokens/view-tokens/)
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/sql.md:189]: To store secrets, you need your authentication token and organization ID:
[View...

File: content/v2.0/query-data/flux/sql.md [hunk 6]
```diff
@@ -154,6 +155,78 @@ Use the variable to manipulate queries in your dashboards.
...
+influx secret update -k <secret-key>
+
+# Example
+influx secret update -k POSTGRES_PASS
+```
+
+**When prompted, enter your secret value.**
+
+{{% warn %}}
+You can provide the secret value with the `-v`, `--value` flag, but the plain text
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/sql.md:203]: would that be `influx secret update -v <secret-key>`?

File: content/v2.0/query-data/flux/sql.md [hunk 7]
```diff
@@ -154,6 +155,78 @@ Use the variable to manipulate queries in your dashboards.
 
 ---
 
+## Use secrets to populate connection credentials
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/sql.md:158]: Use secrets to store SQL database credentials

File: content/v2.0/reference/flux/stdlib/experimental/bigtable/from.md [hunk 8]
```diff
@@ -52,3 +52,26 @@ _**Data type:** String_
 The name of the Cloud Bigtable table to retrieve data from.
 
 _**Data type:** String_
+
+## Examples
+
+{{% note %}}
+The example below use [InfluxDB secrets](/v2.0/security/secrets/) to populate
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/bigtable/from.md:59]: below uses

File: content/v2.0/query-data/flux/sql.md [hunk 9]
```diff
@@ -154,6 +155,78 @@ Use the variable to manipulate queries in your dashboards.
...
+**When prompted, enter your secret value.**
+
+{{% warn %}}
+You can provide the secret value with the `-v`, `--value` flag, but the plain text
+secret may appear in your shell history.
+{{% /warn %}}
+{{% /tab-content %}}
+{{< /tabs-wrapper >}}
+
+### Use secrets in your query
```
Comment (kelseiv) [inline content/v2.0/query-data/flux/sql.md:209]: I notice we have this "Use secrets in your query" on multiple pages. Perhaps we can link to?

**Pull request merged**